### ISSUE TITLE
New enable_tlmgr = true flag does not work.

### ISSUE THREAD

**Issue by ezickler** [r-darwish/topgrade#439]
**What did you expect to happen?**
Uncomment the new "enable_tlmgr = true" line in the config to enable tlmgr update.

**What actually happened?**

```console
  2020-06-19T19:28:09.437Z ERROR topgrade::config > failed to load configuration: unknown field `enable_tlmgr`, expected one of `pre_commands`, `commands`, `git_repos`, `predefined_git_repos`, `disable`, `remote_topgrades`, `ssh_arguments`, `git_arguments`, `tmux_arguments`, `set_title`, `assume_yes`, `yay_arguments`, `no_retry`, `run_in_tmux`, `cleanup`, `notify_each_step`, `accept_all_windows_updates`, `only`, `composer`, `brew`, `linux`, `git`, `windows`, `vagrant` at line 1 column 1
 ```

**Additional details:**
- Which operating system or Linux distribution are you using?
Fedora 32
- How did you install Topgrade?
crates.io via cargo  version 4.7.1
- If a specific step is failing, what happens when you run the command on your own? (Run `topgrade --dry-run` to see which commands Topgrade is running)

<details><pre>
Paste the output of `topgrade -v`
</pre></details>

**Comment by r-darwish** [comment]
Did you put this flag under the [linux] section?

**Comment by ezickler** [comment]
Yes i did. I even deleted my old config and used the template from the new release. Copied from the github repo.

I only disabled some things and set no_retry. Like this.

```
# Disable specific steps - same options as the command line flag
disable = ["gitrepos", "emacs","go","gem"]

# Run specific steps - same options as the command line flag
#only = ["system", "emacs"]

# Do not ask to retry failed steps (default: false)
no_retry = true
 
...

#[linux]
# Arguments to pass yay when updating packages
#yay_arguments = "--nodevel"
#trizen_arguments = "--devel"
enable_tlmgr = true
emerge_sync_flags = "-q"
emerge_update_flags = "-uDNa --with-bdeps=y world"

**Comment by ezickler** [comment]
Oh. Do i need to uncomment the [linux] line?

**Comment by r-darwish** [comment]
Noted. Fixed in f386a3675400482c744b57cc6be2aac43003d3d2
